Library/Formula/clang-format.rb

```diff
@@ -1,18 +1,18 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "http://clang.llvm.org/docs/ClangFormat.html"
-  version "2015-09-01"
+  version "3.7.1"
```

Comment:
Thanks for the explanation and suggestion. I changed it back to a date as this seems to be the simplest solution.